public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
-  return 'Hello, World!';
+function buildProduct(product) {
```

Comment:
You should definitely read Clean Code. It's one of the best books I've ever read. It's all about how important writing clean code is. Not just for other people so they understand your code better, it's also about being able to understand your code better yourself on a later stage, it's a lot easier to maintain and refactor, it gives you something to be proud of, ...
Clean Code is just awesome.
The added value here is that you see buildproducts first. In buildproducts, you see buildproduct, which is just underneath the buildproducts method, which completely makes sense. If you would do it the other way around,  like you do now, it doesn't read that easy because you have to remember that buildproduct is at the top instead of just reading the file top down. The file's still small right now, but once it gets bigger, you'll regret it.